packages/components/molecules/f-mega-modal/README.md

```diff
@@ -17,7 +16,7 @@
 
 ## Usage
 
-### Installation
+## Installation
```

Comment:
This looks correct to me as-is. Line 18 is `h2`, and these are then `h3`?

Reply:
Yup.  The **Usage** being a level two confused me.  I've made it level 1.

Reply:
There's already an `h1` up on line 3, so I think it was still correct previously